client/cvra_bootloader/utils.py

```diff
@@ -110,7 +115,13 @@ def open_connection(args):
         conn = can.adapters.SocketCANConnection(args.can_interface)
     elif args.serial_device:
         port = serial.Serial(port=args.serial_device, timeout=0.1)
-        conn = can.adapters.SerialCANConnection(port)
+
+        if args.large_pages:
+            timeout = 5
+        else:
+            timeout = 0.5
```

Comment:
`timeout =  5 if args.large_pages else 0.5` ?

Reply:
Not a huge fan of those python compressed one liners.